packages/ckeditor5-emoji/src/emojiconfig.ts

```diff
@@ -63,6 +63,25 @@ export interface EmojiConfig {
 	 */
 	skinTone?: SkinToneId;
 
+	/**
+	 * The URL from which the emoji definitions should be loaded.
+	 *
+	 * ```ts
+	 *	ClassicEditor
+	 *		.create( editorElement, {
+	 *			plugins: [ Emoji, ... ],
+	 *			emoji: {
+	 *				definitionsUrl: ''
+	 *			}
+	 *		} )
+	 *		.then( ... )
+	 *		.catch( ... );
+	 * ```
+	 *
+	 * @default 'https://cdn.ckeditor.com/ckeditor5/data/emoji/\{version\}/en.json'
```

Comment:
I'm not sure about default values for `definitionsUrl` and `version`: we do not set those values in the editor config. If they are not provided by the integrator, they remain `undefined`.

Reply:
It's still the default here: https://github.com/ckeditor/ckeditor5/pull/17902/files#diff-bcad77073801aff462771b1e2b907a1f3006cea6057f28fa11504ea853cca0c9R235